src/main/java/com/google/alpollo/SongHead.java

```diff
@@ -0,0 +1,39 @@
+package com.google.alpollo;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Id;
+
+@Entity
+public class SongHead {
+  @Id
+  private String id;
+  @Index
+  private String artist;
+  @Index
+  private String name;
+  @Index
+  private String album;
+
+  public SongHead() {
+  }
+
+  public SongHead(String artist, String name, String album) {
+    this.artist = artist;
+    this.name = name;
+    this.album = album;
+    id = artist + name + album;
```

Comment:
please, add comment that you'll do hashing later (or you can do it in this pr as well)